src/js/apis/youtubeApis.js

```diff
@@ -22,3 +22,22 @@ export const findAllBySearchKey = async ({searchKeyword, pageToken = '', maxSize
         }),
     };
 };
+
+export const findAllByVideoIds = async ({videoIds}) => {
```

Comment:
```suggestion
export const findAllByVideoIds = async ({videoIds} = {}) => {
```

이런식으로 표현해주면 실수로 함수의 인자로 객체를 전달하지 않아도 구조분해 할당시에 에러를 방지할 수도 있습니다ㅎㅎ